srp.go

```diff
@@ -130,20 +130,23 @@ func newSRP(serverSide bool, group *Group, xORv, k *big.Int) *SRP {
 		s.k.Set(k)
 	} else {
 		if _, err := s.makeLittleK(); err != nil {
-			return nil
+			return nil, fmt.Errorf("generating little k: %w", err)
 		}
 	}
-	s.generateMySecret()
+	_, err := s.generateMySecret()
+	if err != nil {
+		return nil, fmt.Errorf("generating secret: %w", err)
+	}
```

Comment:
👋🏼 @jpgoldberg I'm curious why we need to call `s.generateMySecret()` before calling `s.makeB()` or `s.makeA()`. Those methods will themselves call `s.generateMySecret()` as part of their execution: https://github.com/CamiloGarciaLaRotta/srp/blob/d2080afd070083afda9374fdc3917e82727af74b/internal.go#L115-L121

Unless I'm missing something, I think we can remove this call to `s.generateMySecret()`. CI did not fail without it